hazelcast/src/main/java/com/hazelcast/monitor/impl/InstantLocalMapStats.java

```diff
@@ -0,0 +1,230 @@
+    }
+
+    public long getPutOperationCount() {
+        return putCount;
+    }
+
+    public void incrementPuts(long latency) {
+        PUT_COUNT_UPDATER.incrementAndGet(this);
+        TOTAL_PUT_LATENCIES_UPDATER.addAndGet(this, latency);
+        MAX_PUT_LATENCY_UPDATER.set(this, Math.max(maxPutLatency, latency));
+    }
+
+    public long getGetOperationCount() {
+        return getCount;
+    }
+
+    public void incrementGets(long latency) {
+        GET_COUNT_UPDATER.incrementAndGet(this);
+        TOTAL_GET_LATENCIES_UPDATER.addAndGet(this, latency);
+        MAX_GET_LATENCY_UPDATER.set(this, Math.max(maxGetLatency, latency));
+    }
+
+    public long getRemoveOperationCount() {
+        return removeCount;
+    }
+
+    public void incrementRemoves(long latency) {
+        REMOVE_COUNT_UPDATER.incrementAndGet(this);
+        TOTAL_REMOVE_LATENCIES_UPDATER.addAndGet(this, latency);
+        MAX_REMOVE_LATENCY_UPDATER.set(this, Math.max(maxRemoveLatency, latency));
+    }
+
+    public long getTotalPutLatency() {
+        return totalPutLatencies;
+    }
+
+    public long getTotalGetLatency() {
+        return totalGetLatencies;
+    }
+
+    public long getTotalRemoveLatency() {
+        return totalRemoveLatencies;
+    }
+
+    public long getMaxPutLatency() {
+        return maxPutLatency;
+    }
+
+    public long getMaxGetLatency() {
+        return maxGetLatency;
+    }
+
+    public long getMaxRemoveLatency() {
+        return maxRemoveLatency;
+    }
+
+    public long getOtherOperationCount() {
+        return numberOfOtherOperations;
+    }
+
+    public void incrementOtherOperations() {
+        NUMBER_OF_OTHER_OPERATIONS_UPDATER.incrementAndGet(this);
+    }
+
+    public long getEventOperationCount() {
+        return numberOfEvents;
+    }
+
+    public void incrementReceivedEvents() {
+        NUMBER_OF_EVENTS_UPDATER.incrementAndGet(this);
+    }
+
+    @Override
+    public JsonObject toJson() {
+        final JsonObject root = new JsonObject();
+
+        root.add("creationTime", creationTime);
+        root.add("getCount", getCount);
+        root.add("putCount", putCount);
+        root.add("removeCount", removeCount);
+        root.add("numberOfOtherOperations", numberOfOtherOperations);
+        root.add("numberOfEvents", numberOfEvents);
+        root.add("lastAccessTime", lastAccessTime);
+        root.add("lastUpdateTime", lastUpdateTime);
+        root.add("totalGetLatencies", totalGetLatencies);
+        root.add("totalPutLatencies", totalPutLatencies);
+        root.add("totalRemoveLatencies", totalRemoveLatencies);
+        root.add("maxGetLatency", maxGetLatency);
+        root.add("maxPutLatency", maxPutLatency);
+        root.add("maxRemoveLatency", maxRemoveLatency);
+
+        return root;
+    }
+
+    @Override
+    public void fromJson(JsonObject json) {
+        this.creationTime = getLong(json, "creationTime", -1L);
+        GET_COUNT_UPDATER.set(this, getLong(json, "getCount", -1L));
```

Comment:
I think with deserialization the object isn't shared yet and you can safely call the normal field. So 
numberOfEvents = getLong("numberOfEvents"). No need to go through field updaters.